netplugin/plugin/netplugin.go

```diff
@@ -220,3 +196,43 @@ func (p *NetPlugin) GetEndpointStats() ([]byte, error) {
 func (p *NetPlugin) InspectState() ([]byte, error) {
 	return p.NetworkDriver.InspectState()
 }
+
+//GlobalFwdModeUpdate update the forwarding mode
+func (p *NetPlugin) GlobalFwdModeUpdate(cfg Config) {
+	var err error
+
+	if p.NetworkDriver != nil {
```

Comment:
we dont allow the fwd mode change if we have any networks created - so endpoints will not be there